src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -61,6 +67,24 @@
 {% endblock %}
 
 {% block panelContent %}
+
+    <h2>Translation Locales</h2>
+
+    <div class="metrics">
+        <div class="metric">
+            <span class="value">{{ collector.locale }}</span>
```

Comment:
When some data is missing, we use a dash (`-`)  instead of an empty string (e.g. -> `<td class="text-right">{{ listener.priority|default('-') }}</td>`). The problem with a white space is that you don't know if it's because of some error or because there's really no data.

In TranslationDataCollector.php you used an empty string when there's no locale, so maybe here we can check if it's empty and display a `-`